xliff/TotalsUI/TotalsUI_pl.xlf

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0"?>
+<xliff xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.0" srcLang="en" trgLang="pl" xmlns="urn:oasis:names:tc:xliff:document:2.0">
+  <file id="47e178126daef4945b4c134f0407ae8e" original="Assets/Localization/Tables/TotalsUI/TotalsUI_en.asset">
+    <group id="c65c29c31738268469819b53eccf4d6a" name="TotalsUI">
+      <unit id="12249624576" name="PlayLevel" xml:space="preserve">
+        <segment state="translated">
+          <source>Play this level?</source>
+          <target>Chcesz zagrać na tym poziomie?</target>
+        </segment>
+      </unit>
+      <unit id="31891550208" name="PlayOrTravelToLevel" xml:space="preserve">
+        <segment state="translated">
+          <source>Play the level or travel to the location on the map?</source>
+          <target>Chcesz zagrać na tym poziomie czy przenieść się do lokalizacji na mapie?</target>
+        </segment>
+      </unit>
+      <unit id="696000868352" name="Play" xml:space="preserve">
+        <segment state="translated">
+          <source>Play</source>
+          <target>Graj</target>
+        </segment>
+      </unit>
+      <unit id="809087692800" name="Travel" xml:space="preserve">
+        <segment state="translated">
+          <source>Travel</source>
+          <target>Przenieś</target>
```

Comment:
Would it be more grammatically accurate to include the reflexive ("się") at the end?